src/components/course-review-card.js

```diff
@@ -68,7 +68,7 @@ CourseReviewCard.propTypes = {
   code: PropTypes.string,
   name: PropTypes.string,
   numReviews: PropTypes.number,
-  overallRating: PropTypes.number,
+  overallRating: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
Since your code uses both numbers and strings, this seemed like the easiest way to resolve the type warnings :).